forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -27,7 +27,8 @@ object ForageSDK : ForageSDKApi {
     override suspend fun tokenizeEBTCard(
         merchantAccount: String,
         bearerToken: String,
-        customerId: String
+        customerId: String,
+        reusable: Boolean
```

Comment:
nit: I know that the compiler will enforce this but it could be nice to make explicit that `reusable` is optional with a default value of true here
```suggestion
        reusable: Boolean = true
```

Reply:
Hi Devin, I tried to do add this, but it produces the error `an overriding function is not allowed to specify default values for its parameters`. 

For this reason, I figured setting it to true by default in the API it implements would be passed on in this function